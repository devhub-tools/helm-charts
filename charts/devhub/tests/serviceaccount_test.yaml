suite: ServiceAccount
templates:
  - serviceaccount.yaml
release:
  name: devhub
  namespace: devhub

tests:
  - it: should create service account by default
    set:
      serviceAccount:
        annotations:
          custom-annotation: "custom-value"
    asserts:
      - matchSnapshot: {}

  - it: should not create service account if create is false
    set:
      serviceAccount:
        create: false
    asserts:
      - hasDocuments:
          count: 0