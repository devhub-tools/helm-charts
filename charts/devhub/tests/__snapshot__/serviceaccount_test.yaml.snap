should create service account by default:
  1: |
    apiVersion: v1
    kind: ServiceAccount
    metadata:
      annotations:
        custom-annotation: custom-value
      labels:
        app.kubernetes.io/instance: devhub
        app.kubernetes.io/managed-by: Helm
        app.kubernetes.io/name: devhub
        app.kubernetes.io/version: v2.0.5
        helm.sh/chart: devhub-2.1.2
      name: devhub
